### ISSUE TITLE
Adjusting GSIConfig settings seems to not work

### ISSUE THREAD

**Issue by FeodorD** [koczadly/csgo-gsi#7]
First, thank you very much for making this wonderful library, it seems to work well.

I was able to monitor player's HP in real time, however it only seems to update once every second or so. Hence my "live" results are about a second delay. 

With the config below I am getting about a 1 second delay between game state updates whereas I would expect an update every 100ms

```
GSIConfig config = new GSIConfig(1337)
                .setTimeoutPeriod(5.0)
                .setBufferPeriod(0.1)
                .setThrottlePeriod(0.1)
                .setHeartbeatPeriod(5.0)
```

Adjusting buffer period and throttle period does not seem to affect anything.

Do you have any insights about this?
Thank you

**Comment by koczadly** [comment]
Heya, sorry to hear you're having an issue with the library. I've just tried it out, and it seems to work for me (though you are missing the setting of data components from your config - I assume that's only an issue in the example you pasted?)

Are you using the latest version, `1.5.0`? I know an older version suffered a bug exactly like what you're describing, but this should've since been fixed. You could also try setting throttle to `0`, a non-zero throttle can sometimes add extra delays.

If that doesn't help let me know and I'll look into this further for you.

**Comment by koczadly** [comment]
There were a few weird discrepancies in the versioning of 1.5.0, it seems there were 2 separate versions (one deployed, one used locally on my pc). I've released `1.6.0` which should fix the issue if you were already using 1.5.0. Let me know how it goes.

**Comment by FeodorD** [comment]
Thank you very much, i will test it tonight and i will let you know
